python/paddle/fluid/layers/detection.py

```diff
@@ -2221,3 +2222,59 @@ class number
     output.stop_gradient = True
 
     return output
+
+
+def collect_fpn_proposals(multi_rois,
+                          multi_scores,
+                          min_level,
+                          max_level,
+                          post_nms_topN,
+                          name=None):
+    """
+    Collect multi-level rois and select N rois with respect to multi_scores.
```

Comment:
```suggestion
    Collect multi-level RoIs (Region of Interest) and select N RoIs with respect to multi_scores.
```
the following "rois" ==> "RoIs" ,  to avoid confusion

Reply:
Done